{"hierarchy":{"paths":[["doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser","doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ExitCode"]]},"abstract":[{"type":"text","text":"An exit code that indicates successful completion of a command."}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/argumentparser\/exitcode\/success"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"static"},{"text":" ","kind":"text"},{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"text":"success","kind":"identifier"},{"kind":"text","text":": "},{"identifier":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ExitCode","preciseIdentifier":"s:14ArgumentParser8ExitCodeV","kind":"typeIdentifier","text":"ExitCode"}]}],"kind":"declarations"}],"metadata":{"symbolKind":"property","title":"success","role":"symbol","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"success","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:14ArgumentParser8ExitCodeV","text":"ExitCode"}],"externalID":"s:14ArgumentParser8ExitCodeV7successACvpZ","roleHeading":"Type Property","modules":[{"name":"ArgumentParser"}]},"identifier":{"url":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ExitCode\/success","interfaceLanguage":"swift"},"kind":"symbol","references":{"doc://com.SwiftyLab.HelperCoders/documentation/ArgumentParser/ExitCode/success":{"identifier":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ExitCode\/success","type":"topic","title":"success","kind":"symbol","url":"\/documentation\/argumentparser\/exitcode\/success","role":"symbol","fragments":[{"kind":"keyword","text":"static"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"success","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:14ArgumentParser8ExitCodeV","text":"ExitCode","kind":"typeIdentifier"}],"abstract":[{"type":"text","text":"An exit code that indicates successful completion of a command."}]},"doc://com.SwiftyLab.HelperCoders/documentation/ArgumentParser":{"kind":"symbol","role":"collection","url":"\/documentation\/argumentparser","abstract":[],"title":"ArgumentParser","identifier":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser","type":"topic"},"doc://com.SwiftyLab.HelperCoders/documentation/ArgumentParser/ExitCode":{"url":"\/documentation\/argumentparser\/exitcode","identifier":"doc:\/\/com.SwiftyLab.HelperCoders\/documentation\/ArgumentParser\/ExitCode","role":"symbol","fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"ExitCode","kind":"identifier"}],"kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"ExitCode"}],"type":"topic","title":"ExitCode","abstract":[{"text":"An error type that only includes an exit code.","type":"text"}]}}}